### PR TITLE
improve lots of things. colors, rates, params

### PR DESCRIPTION
**Pull request by bkudria**

### CODE REVIEW

File: Breather.pde [hunk 1]
```diff
@@ -1,28 +1,84 @@
...
-    double breath = norm(-exp(breather.getValuef()), -1/E, -E);
-    double brightness = breath * 80;
-    double hue        = 50;
+    double[] breaths = new double[lx.total];
+    double maxBreath = 0;
+
+    for (int p = 0; p < lx.total; p++) {
+      breaths[p] = norm(-exp(breathers[p].getValuef()), -1/E, -E);
+
+      if (Double.compare(maxBreath, breaths[p]) < 0) {
```
Comment (l8on) [inline Breather.pde:68]: @bkudria Is this for better precision?

File: Breather.pde [hunk 2]
```diff
@@ -1,28 +1,84 @@
...
+
   public void run(double deltaMs) {
     breathe();
   }
 
   public void breathe() {
-    double breath = norm(-exp(breather.getValuef()), -1/E, -E);
-    double brightness = breath * 80;
-    double hue        = 50;
+    double[] breaths = new double[lx.total];
```
Comment (l8on) [inline Breather.pde:62]: Do you need a whole new array every run cycle? Would it be enough to just keep track of the current breathe?

**Comment by l8on** [comment]
Probably should have added the comments before I merged, but whatever, it looks great.
Thanks so much @bkudria!

File: Breather.pde (hunk 1 above)
Comment (bkudria) [inline Breather.pde:68]: No, the precision here is the same. not sure what you mean.

File: Breather.pde (hunk 2 above)
Comment (bkudria) [inline Breather.pde:62]: Well, I have an array of breathers, one for every pixel. There is no single "breath" any more.

File: Breather.pde (hunk 1 above)
Comment (l8on) [inline Breather.pde:68]: I meant, as opposed to using the usual `<` operator.

File: Breather.pde (hunk 2 above)
Comment (l8on) [inline Breather.pde:62]: You use `breaths` to find a max value of a function of the `breather` values. You could use a single variable, or test the value inline as opposed to creating this `breaths` array.

File: Breather.pde (hunk 1 above)
Comment (bkudria) [inline Breather.pde:68]: Ah, I was cargo-culting off of equality comparison not working on floats.

File: Breather.pde (hunk 2 above)
Comment (bkudria) [inline Breather.pde:62]: Ah, I see. Will send another PR to fix.
